web/src/giphy.js

```diff
@@ -206,12 +206,23 @@ function toggle_giphy_popover(target) {
                 $popper.on("keydown", ".giphy-gif", ui_util.convert_enter_to_click);
                 $popper.on("keydown", ".compose_gif_icon", ui_util.convert_enter_to_click);
 
+                const $searchInput = $("#giphy-search-query");
+                const $clearButton = $("#giphy_search_clear");
+                $(document).on("input", "#giphy-search-query", () => {
+                    if ($searchInput.val().trim() === "") {
```

Comment:
Let's keep this event handler scoped to `$popper` like other event handlers. Also, similar comment about using snake_case here.